### PR TITLE
Fix issue #13 and #5

### PR DESCRIPTION
**Pull request by andya9**
I used linter-tidy to fix the issue.

### CODE REVIEW

**Comment by andya9** [comment]
Strange thing: if I have an unclosed element called block (<block) it will still popup an error when typing >, saying that value of start is undefined.

File: lib/linter-xmllint.coffee [hunk 1]
```diff
@@ -2,18 +2,20 @@ linterPath = atom.packages.getLoadedPackage("linter").path
 Linter = require "#{linterPath}/lib/linter"
 
 class LinterXmllint extends Linter
-  # ConfigObserver.includeInto(this)
-
   # The syntax that the linter handles. May be a string or
   # list/tuple of strings. Names should be all lowercase.
-  @syntax: 'text.xml'
+  @syntax: ['text.xml']
```
Comment (steelbrain) [inline lib/linter-xmllint.coffee]: Your pull requests conflict with each other https://github.com/AtomLinter/linter-xmllint/pull/15/files#diff-6d82e74b8f5b4b39449abb9a90a920d3R9

**Comment by andya9** [comment]
@steelbrain Fixed here (I thought it was better to keep them separate and merge them one after the other.)

**Comment by andya9** [comment]
By the way, the error when there is a block element is popped up by bracket-matcher, not linter-xmllint; so this pull request should be fine

**Comment by steelbrain** [comment]
Gonna test it locally

**Comment by steelbrain** [comment]
I just tested it locally and it works, funny thing it was working before too. But anyways, You've added some more syntax scopes and stuff. So Looks good to me.

**Comment by andya9** [comment]
Thank’you @steelbrain 
Actually it seems the faulty package is bracket-matcher, but only when this linter-xmllint is enabled; I’ve told them now (issue 137).

**Pull request merged**